### PR TITLE
Cable GH

### PR DESCRIPTION
**Pull request by rickyaristio**
-Add Cable GH
-Add prestress for membrane on Shell GH
-Minor fix on Model IGA GH

### CODE REVIEW

File: Cocodrilo/Cocodrilo_GH/PreProcessing/IO/Model_IGA_GH.cs [hunk 1]
```diff
@@ -134,8 +134,6 @@ protected override void SolveInstance(IGH_DataAccess DA)
                             {
                                 ud = new Cocodrilo.UserData.UserDataEdge();
                                 edge.Key.UserData.Add(ud);
-
-                                mCurveList.Add(edge.Key);
```
Comment (tteschemacher) [inline Cocodrilo/Cocodrilo_GH/PreProcessing/IO/Model_IGA_GH.cs:138]: why do you need to delete this here?

**Comment by tteschemacher** [comment]
generally good.
But it doesn't work yet, right? Didn't you have additional changes within the connection checks?

File: Cocodrilo/Cocodrilo_GH/PreProcessing/IO/Model_IGA_GH.cs (hunk 1 above)
Comment (tteschemacher) [reply Cocodrilo/Cocodrilo_GH/PreProcessing/IO/Model_IGA_GH.cs:138]: oh. ok I see.

**Review by tteschemacher — APPROVED** [review]
Seems good. Thanks for the updates!

**Comment by rickyaristio** [comment]
> generally good. But it doesn't work yet, right? Didn't you have additional changes within the connection checks?

By the way, what do you mean by the connection checks?

**Comment by rickyaristio** [comment]
> Seems good. Thanks for the updates!

Thanks for your help!

**Comment by rickyaristio** [comment]
> > > generally good. But it doesn't work yet, right? Didn't you have additional changes within the connection checks?
> > 
> > 
> > By the way, what do you mean by the connection checks?
> 
> I mean within the GeometryUtilities when geometry intersections are checked.

I see... 
Apparently, I don't need to change anything within the GeometryUtilities. This already works.

**Comment by tteschemacher** [comment]
> I see... Apparently, I don't need to change anything within the GeometryUtilities. This already works.

nice

**Pull request merged**